devimint/src/cli.rs

```diff
@@ -239,7 +239,12 @@ pub async fn handle_command(cmd: Cmd, common_args: CommonArgs) -> Result<()> {
                                 .await?
                                 .send_to(address, client_pegin_amount)
                                 .await?;
-                            Ok(operation_id)
+                            dev_fed.bitcoind().await?.mine_blocks_no_wait(11).await?;
```

Comment:
Doing all that mining in parallel we might even catch some race conditions :rofl: